docs/src/operate/c8y/supported-operations.md

```diff
@@ -140,34 +140,36 @@ The `Supported Operations API` can also be used to add support for custom operat
 For e.g: Cumulocity supports the `c8y_Command` that enables execution of shell commands on the device from the cloud.
 The same operation files can be used to define how the mapper should handle these operations when triggered from the cloud.
 
-### Adding new custom operation
+### Creating new custom operation
+
+Custom operations can be delivered by SmartRest message or C8Y JSON depending on the content of the operation file. The former is limited to SmartRest input. The latter moves the actual execution of the operation to `tedge-agent` where is executed by the workflow.
```

Comment:
If the input is JSON over MQTT, user can choose where the operation should be executed. Either by c8y-mapper ("drop 1") or workflow ("drop 2"). So, the current sentence is half accurate. Also, we should say that SmartREST and the "drop 1" are limited to the main device, but the "drop 2" is for all devices.

Reply:
Resolved in 46b39e8121774672f354a4ec86017bdb715254c3